samples/meetings-content-bubble/csharp/README.md

```diff
@@ -13,13 +13,13 @@ extensions:
 urlFragment: officedev-microsoft-teams-samples-meetings-content-bubble-csharp
 ---
 
-# Meetings Content Bubble
+# Meeting Targeted Notification
 
-This sample illustrates how to implement [Content Bubble](https://docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/create-apps-for-teams-meetings?view=msteams-client-js-latest&tabs=dotnet#notificationsignal-api) In-Meeting Experience.
+This sample illustrates how to implement [Content Bubble](https://docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/create-apps-for-teams-meetings?view=msteams-client-js-latest&tabs=dotnet#notificationsignal-api) In-Meeting Experience. This app also showcase sending targeted notification to meeting participants.
```

Comment:
Minor: I see [ContentBubble]. Let's update it to be '[Targeted In-Meeting Notification]' And Let's also update the link to the feature. This is the new one: https://review.learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#targeted-meeting-notification-api

Reply:
Updated this